CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ else()
   set(NB_MASTER_PROJECT OFF)
 endif()
 
+option(NB_CREATE_INSTALL_RULES "Create installation rules" ${NB_MASTER_PROJECT})
+option(NB_USE_SUBMODULE_DEPS   "Use submodule'd dependencies" ON)
```

Comment:
"Use submodule'd dependencies" -> `Use the nanobind dependencies shipped as a git submodule of this repository`

Reply:
Fixed. Not sure how/if you want to format that line, now that it's quite a bit longer than the rest of the other CMake lines. It looks like ~80 columns is the style.

Reply:
no big deal.